--- /etc/init.d/zimbra.bak      2012-07-30 14:33:11.070218088 +0700
+++ /etc/init.d/zimbra  2012-07-28 15:12:34.361981866 +0700
@@ -62,8 +62,12 @@
     fi
                ;;
        reload|status)
-               command $1
-               RETVAL=$?
+                if [ -f /opt/zimbra/bin/zmcontrol ]; then
+                       command $1
+                       RETVAL=$?
+                else
+                        RETVAL=3
+                fi
                ;;
        *)
                echo $"Usage: $0 {start|stop|restart|reload|status}"
